hadoop-plugin/src/test/groovy/com/linkedin/gradle/azkaban/AzkabanHelperTest.groovy

```diff
@@ -31,4 +34,14 @@ class AzkabanHelperTest {
   public void TestGradleException() {
     AzkabanHelper.fetchSortedFlows(new JSONObject("{\"project\" : \"test-azkaban\",   \"projectId\" : 192,   \"flows\" : [ ] }"));
   }
+
+  @Rule
+  public ExpectedException expectedEx = ExpectedException.none();
+
+  @Test(expected = Exception.class)
```

Comment:
Wow, I dislike that you have to create a `@Rule ExpectedException` so you can verify the exception message in JUnit. In TestNG, the `@Test` annotation directly supports an `expectedExceptionsMessageRegExp` parameter.

It's also weird to me that you first initialize `expectedEx` to `none()` and then tweak it in the test itself. But I see this is what the JUnit Javadoc's example does, too: https://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html

Reply:
It is a bit weird, but the tests are using JUnit right now.